src/app/admin/store/selectors/admin-book.selectors.ts

```diff
@@ -0,0 +1,19 @@
+import { createFeatureSelector, createSelector } from '@ngrx/store';
+import { AppState } from '../../../store/state/app.state';
+import { State } from '../reducers';
+import { AdminBookState } from '../state/admin-books.state';
+
+export const selectAdmin = createFeatureSelector<AppState, State>('admin');
+export const selectBooks = createSelector (
+  selectAdmin,
+  (adminState: State) => adminState.book,
+);
+export const selectBookList = createSelector (
```

Comment:
unused selector
do you need it or it can be removed?

Reply:
fixed